.ruby-version

```diff
@@ -0,0 +1 @@
+2.2.0
```

Comment:
I just refactored the project to use both font-awesome and html_safe. please check the latest commit.